<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>update_query</class>
 <widget class="QWidget" name="update_query">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>update_query</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>110</y>
     <width>189</width>
     <height>149</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="normal">
      <property name="text">
       <string>값 수정</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="server_ma">
      <property name="text">
       <string>서버관리자 자동 할당</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="supervise">
      <property name="text">
       <string>서버 요일 일괄 변경</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="updatetme">
      <property name="text">
       <string>게임 종료 서버 자동 시간 갱신</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox"/>
    </item>
    <item>
     <widget class="QPushButton" name="exec">
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>실 행</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>411</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>270</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
